ToolBox/QuandlBitfinexDownloader/Program.cs

```diff
@@ -38,11 +38,11 @@ static void Main(string[] args)
             {
                 // Load settings from config.json
                 var dataDirectory = Config.Get("data-directory", "../../../Data");
-                bool useDivisor = bool.Parse(Config.Get("bitfinex-use-divisor", "false"));
+                int divisor = int.Parse(Config.Get("bitfinex-divisor", "100"));
```

Comment:
in the future, use the `Config.GetInt` [method](https://github.com/QuantConnect/Lean/blob/master/Configuration/Config.cs#L153) as it is a little cleaner and also accepts the default value. It also will log when default values are being used as an indication to the user that it is a configurable setting